net/snort3/patches/100-remove-HAVE_HS_COMPILE_LIT-to-work-around-upstream-b.patch

```diff
@@ -0,0 +1,33 @@
+Hack: fix hyperscan
+
+Workaround until upstream bug is fixed[1]
+Credit to amish in the AUR package[2]
+
+1. https://github.com/intel/hyperscan/issues/388
+2. https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=snort-nfqueue&id=fbb1f40bf54b3118b56001df6bdb3ba238debee7#n49
```

Comment:
This should be done as proper git patch - author, description, etc.

Reply:
Fixed.